Hardware/Tessel_2_Overview.md

```diff
@@ -0,0 +1,84 @@
+
+The two chips are connected by a SPI bridge that also includes the onboard flash (the [readme for Tessel 2’s firmware repo goes into more detail here](https://github.com/tessel/v2-firmware/blob/master/README.md)).
+
+The whole system is powered from the single Micro USB (device) port.
+
+## Power architecture of Tessel 2
+
+The below table explains how the major components of Tessel 2 are powered.
+
+Component          | Powered by       | Enabled or reset by
+-------------------|------------------|-------------------------------------------------
+5V                 | USB cable        | Presence of USB cable with upstream power
+3.3V Mediatek      | 5V               | GPIO on coprocessor, pull resistor to ENABLE
+Mediatek chip      | 3.3V Mediatek    | GPIO on coprocessor, pull resistor to !RESET
+Flash chip         | 3.3V Mediatek    | GPIO on both the SAM and the Mediatek
+1.8V               | 5V               | 3.3V Mediatek
+RAM for Mediatek   | 1.8V             | Mediatek (involuntary)
+3.3V coprocessor   | 5V               | 5V
+Coprocessor chip   | 3.3V Coprocessor | GPIO on Mediatek, pull resistor to !RESET
+3.3V module port A | 3.3V Coprocessor | GPIO on coprocessor, pull resistor to OFF
+3.3V module port B | 3.3V Coprocessor | GPIO on coprocessor, pull resistor to OFF
+USB hub IC         | 5V               | 3.3V Mediatek
+Power to USB ports | 5V               | Mediatek via USB hub IC, pull resistor to off
+
+What this means in practice is that the coprocessor (SAMD21) has control over both the "power plug" and the reset switch for the Mediatek, while the Mediatek only has control over the SAM's reset line. The rationale here is that the SAM uses a negligible amount of power, but it might, as you suggest, be beneficial to be able to shut down the Mediatek in order to save power.
+
+That said, there is no true "off switch" in the hardware that enables a permanent shutdown so long as external power is applied. Quiescent draw is currently estimated to be <10mW.
+
+With the default firmware in place, triggering a Mediatek reset from the SAM is...discouraged, but results in a clean reboot. I believe the SAM does currently save some state, but it can be reset by the Mediatek, so this isn't really a concern.
+
+Because the flash must be on in order for the Mediatek to boot (that's where [our build of OpenWRT](https://github.com/tessel/openwrt-tessel) lives), it is powered by the 3.3V Mediatek bus. In order for the SAM to program the flash, it must hold the Mediatek in reset. Note that this is most useful for programming the board the first time...and other custom, low-level work, like what you would need to write in order to successfully save state so you could power down the Mediatek and bring it back up again elegantly. The protocol and architecture involved between the Mediatek and the coprocessor is explained more [here](https://github.com/tessel/v2-firmware).
```

Comment:
> the coprocessor is explained more [here](https://github.com/tessel/v2-firmware).

Should the link point to `tessel/t2-firmware` instead of `tessel/v2-firmware`?